tango_ros_common/tango_ros_native/src/tango_ros_node.cpp

```diff
@@ -1118,6 +1118,7 @@ bool TangoRosNode::GetMapName(
     const tango_ros_messages::GetMapName::Request &req,
     tango_ros_messages::GetMapName::Response &res) {
   res.map_name = GetMapNameFromUuid(req.map_uuid);
+  return res.map_name.empty();
```

Comment:
+1, that would indeed make it clear according to http://wiki.ros.org/roscpp/Overview/Services#Class_Methods we should probably always return true.